src/appsco-app.html

```diff
@@ -1339,6 +1356,12 @@
                 company-api="[[ _companyApi ]]">
         </appsco-start-integration>
 
+        <appsco-remove-integration
+                id="appscoRemoveIntegration"
+                integration="[[ integration ]]"
```

Comment:
No need for this if you set "integration" by using: this.$.appscoRemoveIntegration.setIntegration(event.detail.integration);